src/core/analytics.js

```diff
@@ -488,6 +516,26 @@ class Analytics {
       object.executeReadyCallback();
     }
 
+    // Processing cloud mode buffered events
+    const cloudModeEvents = this.store.get('rs_events');
+    if (object.bufferDataPlaneEventsUntilReady && cloudModeEvents && cloudModeEvents.length > 0) {
+      cloudModeEvents.forEach((event) => {
+        const methodName = event[0];
+        event.shift();
+        // convert common names to sdk identified name
+        if (Object.keys(event[0].message.integrations).length > 0) {
+          transformToRudderNames(event[0].message.integrations);
```

Comment:
Why are we doing this? I believe this is already done in the step prior to this.